cli/spark/endpoints/cluster/cluster_ssh.py

```diff
@@ -52,19 +57,23 @@ def execute(args: typing.NamedTuple):
         connect=args.connect
     )
 
-    http_prefix = 'http://localhost:'
     log.info("-------------------------------------------")
     log.info("spark cluster id:    %s", ssh_conf.cluster_id)
     log.info("open webui:          %s%s", http_prefix, ssh_conf.web_ui_port)
     log.info("open jobui:          %s%s", http_prefix, ssh_conf.job_ui_port)
-    log.info("open jobhistoryui:   %s%s", http_prefix, ssh_conf.job_history_ui_port)
-    log.info("open jupyter:        %s%s", http_prefix, ssh_conf.jupyter_port)
-    log.info("open namenodeui:     %s%s", http_prefix, ssh_conf.name_node_ui_port)
-    log.info("open rstudio server: %s%s", http_prefix, ssh_conf.rstudio_server_port)
+    log.info("open jobhistoryui:   %s%s", http_prefix,
+             ssh_conf.job_history_ui_port)
+    log.info("open namenodeui:     %s%s",
```

Comment:
This should no longer be there, should be a plugin. All HDFS, Rstudio and Jupyter ports fall under the plugin category.